server/kit/kitserver.go

```diff
@@ -91,34 +91,38 @@ func NewServer(svc Service) *Server {
 
 	projectID := googleProjectID()
 	if projectID != "" {
-		svcName, svcVersion := "default", "v1.0"
+		svcName, svcVersion := "", ""
 		if isGAE() {
 			_, svcName, svcVersion = getGAEInfo()
 		}
 		if n, v := os.Getenv("SERVICE_NAME"), os.Getenv("SERVICE_VERSION"); n != "" {
```

Comment:
Do we want this to be overwritten if also supplied within GAE? Seems like it'd be a weird side effect. Mind tossing this in an `else if`?